WebHostLib/misc.py

```diff
@@ -0,0 +1,170 @@
+import datetime
+import os
+
+import jinja2.exceptions
+from flask import request, redirect, url_for, render_template, Response, session, abort, send_from_directory
+
+from models import count, Seed, commit, Room, db_session, Command, UUID, uuid4
```

Comment:
```suggestion
from .models import count, Seed, commit, Room, db_session, Command, UUID, uuid4
```
i get an import error otherwise